src/components/ppom/transactions.js

```diff
@@ -228,18 +206,15 @@ export function ppomMaliciousTransactionsAndSignatures(parentContainer) {
 
   // Malicious Contract interaction
   maliciousContractInteractionButton.onclick = async () => {
-    const contractAddress =
-      MALICIOUS_CONTRACT_ADDRESSES[globalContext.networkName] ||
-      MALICIOUS_CONTRACT_ADDRESSES.default;
```

Comment:
we directly use the malicious contract interaction data from the sample contracts. We don't want to hide this button (unsure why this was added)
The payload for the hex data is what is malicious, no matter to which address we point (in this case, Blockaid flags the tx as malicious because of the hex data, not checking the to address -- which makes our life easier)